examples/dots.py

```diff
@@ -0,0 +1,51 @@
+from threading import Thread
+from time import sleep
+
+from nuimo import Controller, ControllerManager, ControllerListener, LedMatrix
+
+
+MAC_ADDRESS = "c4:d7:54:71:e2:ce"
```

Comment:
Shall we provide a `__main__` entry point that takes the MAC address as an argument?